miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java

```diff
@@ -675,13 +677,14 @@ private boolean updateSequencingParameters(final Run target, User user, Predicat
     return false;
   }
 
-  private boolean updateContainerFromNotification(final Run target, User user, int laneCount, String containerSerialNumber,
-      final Instrument sequencer, final GetLaneContents getLaneContents) throws IOException {
+  private boolean updateContainerFromNotification(final Run target, User user, SequencingContainerModel containerModel,
+      String containerSerialNumber, final Instrument sequencer, final GetLaneContents getLaneContents) throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "sequencer". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)